src/types/relocatable.rs

```diff
@@ -128,9 +128,9 @@ impl Relocatable {
         Ok(relocatable!(self.segment_index, new_offset))
     }
 
-    pub fn add(&self, other: usize) -> Result<Self, VirtualMachineError> {
+    pub fn add(&self, other: usize) -> Self {
```

Comment:
This means we can `impl Add` on this :)

Reply:
it was already implemented here:
https://github.com/lambdaclass/cairo-rs/blob/main/src/types/relocatable.rs#L67-L72